jina/parsers/peapods/runtimes/remote.py

```diff
@@ -5,27 +5,80 @@
 from ....enums import CompressAlgo
 
 
-def mixin_remote_parser(parser):
+def mixin_remote_runtime_parser(parser):
     """Add the options for remote expose
     :param parser: the parser
     """
-    gp = add_arg_group(parser, title='Expose')
+    gp = add_arg_group(parser, title='RemoteRuntime')
+    _add_host(gp)
 
     gp.add_argument(
-        '--host',
-        type=str,
-        default=__default_host__,
-        help=f'The host address of the runtime, by default it is {__default_host__}.',
+        '--port-jinad',
+        type=int,
+        default=12345,  # helper.random_port(),
+        help='The port of the remote machine for usage with JinaD.',
     )
 
+
+def mixin_remote_jinad_parser(parser):
+    """Add the options for remote expose
+    :param parser: the parser
+    """
+    gp = add_arg_group(parser, title='RemoteJinad')
+    _add_host(gp)
+
     gp.add_argument(
-        '--port-expose',
+        '--port',
+        type=int,
+        default=helper.random_port(),
+        help='The port of the host exposed for connecting to.',
+    )
+
+
+def mixin_client_gateway_parser(parser):
+    """Add the options for remote expose
+    :param parser: the parser
+    """
+    gp = add_arg_group(parser, title='ClientGateway')
+    _add_host(gp)
+    _add_proxy(gp)
+
+    gp.add_argument(
+        '--port-gateway',
         type=int,
         default=helper.random_port(),
-        help='The port of the host exposed to the public',
+        help='The port of the Gateway, which the client should connect to.',
     )
 
+
+def mixin_grpc_gateway_parser(parser):
```

Comment:
Except `proxy` arg. this can be called `mixin_gateway_parser`?

Reply:
What do you mean with "Except `proxy` arg"?

Reply:
Got. it. renamed the function.